ruby_on_rails/rails_basics/deployment.md

```diff
@@ -187,9 +187,9 @@ The application logs are the output of your application as it's running. It tell
 
 So if you're getting a 500 error, you can open up the logs and watch them closely as you refresh the page on your app in the browser to reproduce the error. This will either tell you exactly what the problem is or generate some leads you can use to dig further.
 
-If the logs tell you, the application crashes before it even boots and does not give you anything more to go on. The next tool to reach for is the Rails console. The Rails console boots your app, so just opening it in production can produce a better error message that will tell you exactly why the app is crashing and what part of the app is causing it.
+If the logs tell you the application crashes before it even boots and do not give you anything more to go on, the next tool to reach for is the Rails console. The Rails console boots your app, so just opening it in production can produce a better error message that will tell you exactly why the app is crashing and what part of the app is causing it.
```

Comment:
Do doesn't read well here. 

perhaps something like this instead:
```suggestion
If the logs tell you the application crashes before it even boots and gives you nothing else to go on, the next tool to reach for is the Rails console. The Rails console boots your app, so just opening it in production can produce a better error message that will tell you exactly why the app is crashing and what part of the app is causing it.
```

Reply:
I like your suggestion. I think it'd need to be "give" instead of "gives" since we're treating "the logs" as plural. It's clearer if I remove the first part of the phrase:
"If the logs give you nothing else to do on …"

Reply:
Sounds good, thanks @rhodes-david 💪